.github/workflows/dockerimage.yml

```diff
@@ -22,11 +22,15 @@ jobs:
         login-server: healthplatformregistry.azurecr.io
         username: ${{ secrets.REGISTRY_USERNAME }}
         password: ${{ secrets.REGISTRY_PASSWORD }}
-    
+
+    - name: get-npm-version
+      id: package-version
+      uses: martinbeentjes/npm-get-version-action@master
+
     - name: Build the Docker image
       env:
         IMAGE_NAME: healthplatformregistry.azurecr.io/fhirconverter
       run: |
-        docker build . -t $IMAGE_NAME:${{ github.sha }} -t $IMAGE_NAME:$(date +'%Y-%m-%d') -t $IMAGE_NAME:latest
+        docker build . -t $IMAGE_NAME:${{ github.sha }} -t $IMAGE_NAME:v${{ steps.package-version.outputs.current-version }}-$(date +'%Y-%m-%d') -t $IMAGE_NAME:latest
```

Comment:
How about removing date from tag here (keeping only vmajor.minor.patch format), and adding the version to available list only when it's stable (after switching to next minor/patch version)?

Reply:
Agree. This version syntax is more semantic and powerful. In this way, we should increment the version number in package.json during each update, right?

Reply:
Only thing we need to take care of is to increment version (major or minor or patch) after adding previous one in available list. Increment version after each update (internal version) doesn't impact customers, so it's not a strict requirement.

Reply:
Sounds good. Thanks!